libs/service-portal/finance/src/lib/paths.ts

```diff
@@ -0,0 +1,12 @@
+export enum FinancePaths {
+  FinanceRoot = '/fjarmal',
+  FinanceStatus = '/fjarmal/stada',
+  FinanceTransactions = '/fjarmal/faerslur',
+  FinanceEmployeeClaims = '/fjarmal/laungreidendakrofur',
+  FinanceLocalTax = '/fjarmal/utsvar',
+  FinanceBills = '/fjarmal/greidslusedlar-og-greidslukvittanir',
+  FinanceVehicles = 'https://mitt.samgongustofa.is/',
```

Comment:
Finance vehicles ? 🤔  what is dis ?

Reply:
Not really sure, it's before my time and not being used. I'll remove it 😄